play-json/src/main/resources/reference.conf

```diff
@@ -3,3 +3,14 @@ akka.actor {
     "com.lightbend.lagom.scaladsl.playjson.PlayJsonSerializer" = 1000004
   }
 }
+
+
+lagom.serialization.json {
+
+  # The serializer will compress the payload if the message class
+  # was registered using JsonSerializer.compressed and the payload
+  # is larger than this value. Only used for remote messages within
```

Comment:
You are right, this is applicable not only to messages on the wire (inter node message passing), but also for persistence.